src/applications/ivc-champva/shared/components/fileUploads/upload.js

```diff
@@ -23,9 +23,17 @@ export function createPayload(file, _formId, password) {
 }
 
 export function findAndFocusLastSelect() {
-  const lastSelect = [...document.querySelectorAll('select')].slice(-1);
+  const lastSelect = [...document.querySelectorAll('va-select')].slice(-1);
   if (lastSelect.length) {
     focusElement(lastSelect[0]);
+  } else {
+    // focus on upload button as a fallback
+    focusElement(
+      // including `#upload-button` because RTL can't access the shadowRoot
+      'button, #upload-button',
+      {},
+      document.querySelector(`#upload-button`)?.shadowRoot,
+    );
   }
   return lastSelect;
 }
```

Comment:
Previously was just focusing on the `<select>` box if present, however for file uploads where it's not focus was being returned to the top of the page. This sets focus on the upload button if a `<va-select>` is not present.

Also, since the `<select>` elements are buried in the shadow dom when they _are_ present, setting focus to the containing `va-select` will work more reliably.